app/main/helpers/validation.py

```diff
@@ -282,6 +282,27 @@ def errors(self):
         return errors_map
 
 
+class DOS5Validator(SharedValidator):
+    """Following an accessibility review, a number of questions and answers were changed for DOS 5"""
+    email_validation_fields = {"contactEmailContractNotice", "contactEmail"}
+
+    number_fields = [("subcontracting30DayPayments", 0, 100)]
+
+    def formatting_errors(self, answers):
+        error_map = super(DOS5Validator, self).formatting_errors(answers)
+
+        for field, minimum, maximum in self.number_fields:
+            try:
+                number = float(self.answers.get(field))
```

Comment:
We want users to be able to enter the percentage sign [[1]](https://design-system.service.gov.uk/components/text-input/#prefixes-and-suffixes).